inst/extdata/PA2022CH_de_exec_summary/template.Rmd

```diff
@@ -98,7 +98,7 @@ number_peers <- readr::read_csv(
 
 
-Diese Kurzfassung ist eine Zusammenfassung der Einzelergebnisse Ihres hochgeladenen Portfolios \startunderscoreletter {`r portfolio_name`}\stopunderscoreletter. \startunderscoreletter `r paste0(toupper(substr(investor_name, 1, 1)), substr(investor_name, 2, nchar(investor_name)))` \stopunderscoreletter hat eine Zusammenfassung pro Portfolio erhalten. Ein detaillierter interaktiver Einzelbericht (pro Portfolio oder gruppiert) kann über einen persönlichen Zugang auf platform.transitionmonitor.com in Deutsch, Französisch und teilweise Englisch eingesehen werden. Ein Metabericht mit anonymisierten und aggregierten Daten aller Teilnehmer sowie Sektorberichte sind auf der Website des BAFU \textbf{(\href{https://www.bafu.admin.ch/bafu/de/home/themen/klima/fachinformationen/klima-und-finanzmarkt.html}{link})} publiziert und stehen zum Download bereit.
+Diese Kurzfassung ist eine Zusammenfassung der Einzelergebnisse Ihres hochgeladenen Portfolios \startunderscoreletter {`r portfolio_name`}\stopunderscoreletter. \startunderscoreletter `r paste0(toupper(substr(investor_name, 1, 1)), substr(investor_name, 2, nchar(investor_name)))` \stopunderscoreletter hat eine Zusammenfassung pro Portfolio erhalten. Ein detaillierter interaktiver Einzelbericht (pro Portfolio oder gruppiert) kann über einen persönlichen Zugang auf der Transition Monitor Plattform in Deutsch, Französisch und teilweise Englisch eingesehen werden \textbf{\href{https://platform.transitionmonitor.com/}{(link)}}. Ein Metabericht mit anonymisierten und aggregierten Daten aller Teilnehmer sowie Sektorberichte sind auf der Website des BAFU \textbf{(\href{https://www.bafu.admin.ch/bafu/de/home/themen/klima/fachinformationen/klima-und-finanzmarkt.html}{link})} publiziert und stehen zum Download bereit.
```

Comment:
in the 2nd, unchanged link on this line, it uses a different style, i.e. the `(` and `)` are in different positions... not sure if that's a problem

Reply:
no, I saw that, but both are the same